examples/tasks/psutil-statistics-graphite.json

```diff
@@ -17,9 +17,10 @@
       },
       "process": [
```

Comment:
There is an ordinance issue on this task manifest. Publish node should be inside process node:
```JSON
{
  "version": 1,
  "schedule": {
    "type": "simple",
    "interval": "1s"
  },
  "max-failures": 10,
  "workflow": {
    "collect": {
      "metrics": {
        "/intel/psutil/load/load1": {},
        "/intel/psutil/load/load15": {},
        "/intel/psutil/load/load5": {},
        "/intel/psutil/vm/available": {},
        "/intel/psutil/vm/free": {},
        "/intel/psutil/vm/used": {}
      },
      "process": [
        {
          "plugin_name": "statistics",
          "publish": [
            {
              "plugin_name": "graphite",
              "config": {
                "server": "127.0.0.1",
                "port": 2003
              }
            }
          ]
        }
      ]
    }
  }
}
```